### DISCUSSION TITLE
problem of the shared folder of Zelda ocarina of time Spanish and zelda majora's mask Spanish and also a bug (I think) when using the external memory it is not saved in the selected folder to save the game data

### DISCUSSION THREAD

**Discussion by Exard1447** [mupen64plus-ae/mupen64plus-ae#1099] [Q&A]
good morning or afternoon or evening, I don't know if it happens to you but the folder where the games are saved, save state, auto save from zelda ocarina of time spanish is in the zelda majora mask spanish folder in mupen64plus fz pro since the free version doesn't That happens and I have also had problems using the external memory, it is not saved in the selected folder to save the game data and I have to export and import when updating or changing phones. I'm sorry if I don't understand since I'm using Google translator xd

**Comment by fzurita** [comment]
Me imagino que hablas español porque estás tratando de jugar las versiones en español.

En realidad es difícil entender es traducción. Puedes escribir lo que pasa en español?
